src/components/atoms/picker-sheet/index.jsx

```diff
@@ -1,10 +1,12 @@
 import React from 'react';
-import { StyleSheet } from 'react-native';
+import { StyleSheet, Picker } from 'react-native';
 
 // React Native's own picker is deprecated, this is recommended and supported in expo SDK 38
 // Import from @react-native-community/picker/js/Picker to avoid warnings due to
 // https://github.com/react-native-community/react-native-picker/issues/109
-import Picker from '@react-native-community/picker/js/Picker';
+// remove because storybook does not work with it
```

Comment:
```suggestion
// remove because storybook does not transpile the ES6 on import
```

We know what the problem is, so we can share that